src/pages/Delivery/Outside/index.tsx

```diff
@@ -108,25 +143,22 @@ export default function DeliveryOutside() {
       </div>
       <Button
         fullWidth
-        className="mt-auto h-[2.875rem]"
-        onClick={() => {
-          if (address) {
-            setPostAddress({
-              zip_number: address.zip_no,
-              si_do: address.si_nm,
-              si_gun_gu: address.sgg_nm,
-              eup_myeon_dong: address.emd_nm,
-              road: address.rn,
-              building: address.bd_nm,
-              detail_address: '',
-              full_address: '',
-            });
-            navigate('/delivery/outside/detail', { state: { roadAddress } });
-          }
-        }}
+        className="mt-auto mb-5 h-[2.875rem]"
+        onClick={() => handleAddressSelect(address!)}
+        disabled={!address}
       >
         주소 선택
       </Button>
+      {isModalOpen && (
+        <Modal isOpen={isModalOpen} onClose={closeModal}>
+          <div className="flex h-36 w-full flex-col items-center gap-6 px-8 py-6">
+            <p className="text-[15px] leading-[1.6] font-normal text-neutral-600">배달이 불가능한 지역이에요.</p>
```

Comment:
```suggestion
            <p className="text-[15px] leading-[1.6] text-neutral-600">배달이 불가능한 지역이에요.</p>
```
font-normal이 기본값이어서 부모에서 정해진 것이 없다면 빼도 될 것 같아요

Reply:
삭제했습니다😅 59d5ca415b18b3c738a05cf1d2e173e0caec3b14